src/coreclr/debug/daccess/dacdbiimplstackwalk.cpp

```diff
@@ -433,6 +449,20 @@ ULONG32 DacDbiInterfaceImpl::GetCountOfInternalFrames(VMPTR_Thread vmThread)
     ULONG32 uCount = 0;
     while (pFrame != FRAME_TOP)
     {
+#ifdef FEATURE_EH_FUNCLETS
+        if (g_isNewExceptionHandlingEnabled && InlinedCallFrame::FrameHasActiveCall(pFrame))
+        {
+            // Skip new exception handling helpers
+            InlinedCallFrame *pInlinedCallFrame = (InlinedCallFrame *)pFrame;
+            PTR_NDirectMethodDesc pMD = pInlinedCallFrame->m_Datum;
```

Comment:
`m_Datum` is not always initialized. For example, it is not initialized by `JIT_PInvokeBegin/End` helpers. This check does not look reliable.

Reply:
Right, it means that this check won't kick in as expected when we use `JIT_PInvokeBegin` for the PInvoke transition.

Reply:
Ah, I see what you mean. Ignore my comment.